hw/ip/usbdev/doc/_index.md

```diff
@@ -304,71 +278,117 @@ Initially these will be sent to device ID 0.
 When a Set Address request is received, the device ID received must be stored in the {{< regref "usbctrl.device_address" >}} register.
 Note that device 0 is used for the entire control transaction setting the new device ID, so writing the new ID to the register should not be done until the ACK for the Status stage has been received (see [USB 2.0 specification](https://www.usb.org/document-library/usb-20-specification)).
 
+The host will then issue additional control transfers to Endpoint 0 to configure the device, now to the device's configured address.
+In response to the Set Configuration request, software should set up the rest of the endpoints for that configuration, including configuring the flow control behavior for OUT endpoints via the {{< regref "set_nak_out" >}} register, configuring the endpoint type via the {{< regref "rxenable_setup" >}} register (for a control endpoint) and the {{< regref "out_iso" >}} and {{< regref "in_iso" >}} registers (for isochronous OUT and IN endpoints, respectively).
+Finally, software should enable the configured endpoints via the {{< regref "ep_out_enable" >}} and {{< regref "ep_in_enable" >}} registers.
```

Comment:
ah yes, do we document anywhere what happens if ep_*_enable is not set and someone tries to talk to it?
I think this is related to the point we're making above.

Reply:
It's in the register doc for the fields:

For ep_out_enable:
```
                  If the bit is clear, any SETUP or OUT packets sent to the endpoint will be ignored.
```

For ep_in_enable:
```
                  If the bit is clear then any IN packets sent to the endpoint will be ignored.
```